simd/src/Kokkos_SIMD.hpp

```diff
@@ -30,6 +30,20 @@
 
 #include <Kokkos_Macros.hpp>
 
+// FIXME_OPENMPTARGET The device pass disables all compiler macros checked
+#ifdef KOKKOS_ENABLE_OPENMPTARGET
+#if defined(KOKKOS_ARCH_AVX2)
+#include <Kokkos_SIMD_AVX2.hpp>
+#endif
+
+#if defined(KOKKOS_ARCH_AVX512XEON)
+#include <Kokkos_SIMD_AVX512.hpp>
+#endif
+
+#ifdef __ARM_NEON
```

Comment:
how does this help?

Reply:
It just mirrors what we do below as well. We could pull it out but I would expect us to guard this in a similar way in the near future as well.